packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -923,20 +929,21 @@ class DebuggerController extends DisposableController
     }
 
     final variables =
-        frame.vars.map((v) => DartObjectNode.create(v, isolateRef)).toList();
+        frame.vars!.map((v) => DartObjectNode.create(v, isolateRef)).toList();
     variables
       ..forEach(buildVariablesTree)
-      ..sort((a, b) => sortFieldsByName(a.name, b.name));
+      ..sort((a, b) => sortFieldsByName(a.name!, b.name!));
     return variables;
   }
 
   List<Frame> _framesForCallStack(
-    List<Frame> stackFrames, {
-    List<Frame> asyncCausalFrames,
-    InstanceRef reportedException,
+    List<Frame>? stackFrames, {
```

Comment:
instead of allowing stackFrames to be nullable, default to an empty list where this method is called (e.g. `stack.frames ?? []`)